Gemfile

```diff
@@ -52,7 +52,7 @@ group :development, :staging, :test do
 end
 
 group :staging, :production do
-  gem 'mysql', "~> 2.9.0"
+  gem 'mysql2', "~> 0.3"
```

Comment:
Is `~> 0.3` really what we want? I /think/ we're trying to avoid `mysql2 0.4`, which isn't compatible with ActiveRecord 4.x.